packages/edit-site/src/components/site-hub/index.js

```diff
@@ -118,7 +118,13 @@ const SiteHub = memo( ( { isTransparent, className } ) => {
 				>
 					<Button
 						{ ...siteIconButtonProps }
-						className="edit-site-layout__view-mode-toggle"
+						className={ classnames(
+							'edit-site-layout__view-mode-toggle',
+							{
+								'edit-site-site-hub__view-mode-toggle--navigation':
+									siteIconButtonProps.role === 'button',
+							}
+						) }
```

Comment:
Instead of relying on a new `className`, I think it would be better to use the `role="button"` selector in the CSS.

Reply:
Thanks for the update, @jack-lewin 🙇. I've made the changes accordingly.
 Additionally, I too believe that we should add some styling to enhance its visual appeal and ensure easy differentiation. Initially, I considered increasing the spread radius of the box shadow, but that could lead to design inconsistencies. I'm looking for some options that align better with the existing design.

Reply:
For the design, [it was recommended here](https://github.com/WordPress/gutenberg/issues/66375#issuecomment-2443562206):

> Instincts would go towards an inner focus style on the entire square, similar to all other focus styles.